README.md

```diff
@@ -76,153 +44,16 @@ public class Main {
 }
 ```
 
-## Java Client Configuration Parameter
-
-### JVM Parameter
-
-The following includes JVM related parameters.
-
-#### tikv.pd.addresses
-- pd addresses, separated by comma
-- default: 127.0.0.1:2379
-
-#### tikv.grpc.timeout_in_ms
-- timeout of grpc request  
-- default: 600ms
-
-#### tikv.grpc.scan_timeout_in_ms
-- timeout of scan/delete range grpc request
-- default: 20s
-
-#### tikv.importer.max_kv_batch_bytes
-- Maximal package size transporting from clients to TiKV Server (ingest API)
-- default: 1048576 (1M)
-
-#### tikv.importer.max_kv_batch_size
-- Maximal batch size transporting from clients to TiKV Server (ingest API)
-- default: 32768 (32K)
-
-#### tikv.scatter_wait_seconds
-- time to wait for scattering regions
-- default: 300 (5min)
-
-#### tikv.rawkv.default_backoff_in_ms
-- RawKV default backoff in milliseconds
-- default: 20000 (20 seconds)
-
-### Metrics Parameter
-
-#### tikv.metrics.enable
-- whether to enable metrics exporting
-- default: false
-
-#### tikv.metrics.port
-- the metrics exporting http port
-- default: 3140
+Find more demo in [KVRawClientTest](https://github.com/birdstorm/KVRawClientTest/)
```

Comment:
better to move from personal repo to shared repo

Reply:
Agree. How about addressing this issue in the near future?

Reply:
+1

Reply:
create an issue?

Reply:
I created a example section in the document: https://github.com/tikv/client-java/issues/348#issuecomment-992077936

Reply:
The README of [tokio](https://github.com/tokio-rs/tokio) might a great example for library

Reply:
We could add a example in the repository just like `client-go`